src/client/components/localisation/ball/model.ts

```diff
@@ -0,0 +1,32 @@
+import { observable } from 'mobx'
+import { Vector3 } from '../../../math/vector3'
+
+export class BallModel {
+  @observable public name: string
+  @observable public position: Vector3
+  @observable public radius: number
+  @observable public color: string
+
+  public constructor(name: string, position: Vector3, radius: number, color: string) {
+    this.name = name
+    this.position = position
+    this.radius = radius
+    this.color = color
+  }
+
+  public static of(opts: BallModelOpts = {}) {
+    return new BallModel(
+      opts.name || 'Ball',
+      opts.position || Vector3.of(),
+      opts.radius || 0.0335,
+      opts.color || '#FFCC00',
```

Comment:
🚗 🔫  We recently switched to using [this syntax](https://github.com/NUbots/NUsight2/pull/100/commits/3471a7440804f21934ea294bd16e0e13e444eabe) for default values.

Allows us to be consistent for numerical and "falsey" values, e.g. `scale || 1` will actually return `1` if scale has a value of `0`, where as using `{ scale = 1 }` will only default it if `scale === undefined`.